docs/organizations/managing-security-and-risk.md

```diff
@@ -341,6 +341,9 @@ Security and risk management supports checking the languages and infrastructure-
 
 ## Supported security categories
 
+!!! note
+    Due to a recent update, some issues may be temporarily assigned the **Not yet categorized** category. To categorize these issues, you can [reanalyze the default branch of the relevant repository](../faq/repositories/how-do-i-reanalyze-my-repository.md#reanalyzing-a-branch).
```

Comment:
"Due to a recent update" is a bit vague... If we know the cause, any reason for omitting it?
I leave a suggestion for rephrasing, feel free to change it.

Also, I don't see a particular reason to place this admonition before the table, as it doesn't impact any procedure. I see it more as additional information to the "official" list of categories, thus it would fit well after the table. WDYT?

```suggestion
    Issues detected before the Security and risk management launch will be temporarily set with the **Not yet categorized** category. To categorize these issues, [reanalyze the default branch of the relevant repository](../faq/repositories/how-do-i-reanalyze-my-repository.md#reanalyzing-a-branch).
```

Reply:
The issue is that these issues were not detected before SRM was launched, but before we added the new category filters (about a week ago).
This is an internal implementation detail. In my opinion, explaining this to the user would take a very long sentence and wouldn't give much value. I assume the user is not interested in the *why*, but in *what to do* about it.

OK for moving this after the table 👍

Reply:
Fair enough. 😄